webpack_loader/loader.py

```diff
@@ -37,21 +37,22 @@ def get_assets(self):
             return self._assets[self.name]
         return self.load_assets()
 
-    def filter_chunks(self, chunks):
+    def filter_chunks(self, chunks, files):
         for chunk in chunks:
-            ignore = any(regex.match(chunk['name'])
+            ignore = any(regex.match(chunk)
                          for regex in self.config['ignores'])
             if not ignore:
-                chunk['url'] = self.get_chunk_url(chunk)
-                yield chunk
+                url = self.get_chunk_url(chunk, files)
+                yield { 'name': chunk, 'url': url }
 
-    def get_chunk_url(self, chunk):
-        public_path = chunk.get('publicPath')
+    def get_chunk_url(self, chunk, files):
+        chunk_file = files[chunk]
```

Comment:
If all this function is doing with files is to look up for a file using a name, I suggest it receive the file directly and the caller does the lookup.

Reply:
Done, I kept the function signature as it was before.